packages/databyss-editor/interfaces/ranges.ts

```diff
@@ -0,0 +1,2 @@
+import { Range } from './'
+export type Ranges = Array<Range>
```

Comment:
Can we just use `Array<Range>` or `Range[]` in the function signature? Usually you don't need a separate type for the array version of a type...